pkg/apps/apps.jsx

```diff
@@ -52,6 +52,10 @@ const App = () => {
     if (!metainfo_db.ready)
         return <EmptyStatePanel loading />;
 
+    if (path.length > 1) {
+        console.log("this is a long path to walk");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19081-20230706-152338-650ca044-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)